code/model/editableformfields/EditableMultipleOptionField.php

```diff
@@ -171,7 +177,7 @@ public function getHasAddableOptions() {
 	 */
 	protected function getOptionsMap() {
 		$optionSet = $this->Options();
-		$optionMap = $optionSet->map('EscapedTitle', 'Title');
+		$optionMap = $optionSet->map('Value', 'Title');
```

Comment:
Yeah, but for empty options we don't add actual items; We do setEmptyString on the formfield right? In which case, any explicit options should default to some value.